src/components/funding-account/funding-account.edgedb.repository.ts

```diff
@@ -6,5 +6,7 @@ import { FundingAccountRepository } from './funding-account.repository';
 
 @Injectable()
 export class FundingAccountEdgeDBRepository
-  extends RepoFor(FundingAccount).withDefaults()
+  extends RepoFor(FundingAccount, {
+    hydrate: (fa) => fa['*'],
```

Comment:
We can still use splats? I might've misunderstood the PR description! This is nice!

Reply:
We can still use them, just not generically (in RepoFor). Which is why it had to be explicitly added here where we have a concrete FundingAccount type.